app/models/educator.rb

```diff
@@ -17,14 +17,14 @@ class Educator < ActiveRecord::Base
 
```

Comment:
This file is almost all renaming or removing code that was moved into `Authorizer`.